### DISCUSSION TITLE
Possible to add "Reverse Scroll" for Trackpad?

### DISCUSSION THREAD

**Discussion by TomOrangeOnion** [Caldis/Mos#467] [Q&A]
I understand this app is designed for a mouse but is there any chance you can add one more "Reverse Scroll" option for a  trackpad?

I use a mouse when I use MacBook attached to an external monitor, but I really appreciate this reverse function when I use MacBook alone with a touchpad.

Thanks!

**Comment by b11e** [comment]
bump

would like checkboxes or some option to choose like below

<img width="493" height="338" alt="image" src="https://github.com/user-attachments/assets/05a61b29-ec8f-4d60-8577-1538493c7cf5" />
